content/various/FastInput.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * Author: chilli
+ * License: CC0
+ * Source:
+ * Description: Reads in arbitrary amounts of integral values.
+ * Status: tested on SPOJ INTEST
+ * Time: About 3x as fast as cin.
+ * Usage: int a, b[5]; ll c;
+ * read_int(a, b[0], c);
+ */
+#pragma once
+
+struct GC {
+    char buf[1 << 16 | 1];
+    int bc = 0, be = 0;
+    char operator()() {
+        if (bc >= be) {
+            be = fread(buf, 1, sizeof(buf) - 1, stdin);
+            buf[be] = bc = 0;
+        }
+        return buf[bc++];
+    }
+} gc;
```

Comment:
IO/io may be a better name, I confuse GC with garbage collection

Reply:
I use `gc` to stand for `getChar`.